spec/sasl_authenticator_spec.rb

```diff
@@ -25,7 +25,7 @@
 
   let(:sasl_authenticator) {
     Kafka::SaslAuthenticator.new(
-      { logger: logger }.merge(auth_options)
+      **{ logger: logger }.merge(auth_options)
```

Comment:
These changes were needed to support ruby 3 due to https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/.